modules/mtbseq/mtbseq.nf

```diff
@@ -61,12 +61,12 @@ process MTBSEQ {
 
 
 
-params.resultsDir_mtbseq_cohort = "${params.outdir}/mtbseq_cohort"
-params.saveMode_mtbseq_cohort = 'copy'
-params.shouldPublish_mtbseq_cohort = true
+params.results_dir_mtbseq_cohort = "${params.outdir}/mtbseq_cohort"
+params.save_mode_mtbseq_cohort = 'copy'
+params.should_publish_mtbseq_cohort = true
 
 process MTBSEQ_COHORT {
-    publishDir params.resultsDir_mtbseq_cohort, mode: params.saveMode_mtbseq_cohort, enabled: params.shouldPublish_mtbseq_cohort
+    publishDir params.results_dir_mtbseq_cohort, mode: params.save_mode_mtbseq_cohort, enabled: params.should_publish_mtbseq_cohort
```

Comment:
Hi @Mxrcon ,

Perhaps we can also update this pipeline with the same changes as mentioned here 
https://github.com/bioinformatics-lab/mtb_mono_inh_nf/pull/8#pullrequestreview-693639958

Reply:
yes sure! I'll implement that very soon

Reply:
@abhi18av Also: Do you think that we should update the output declaration from `mtbseq_profile` to follow the same pattern of `rita bahia`? or we should keep them differently.

Reply:
`mtbseq_profile` ? 🤔

Reply:
on rita you use:

```
    output:
    path("${genomeFileName}")
```

and on mtb_mono:

```
    output:
    val("${genomeFileName}") // Genome name
    path("""${genomeFileName}_results""") // Folder
    path("""${genomeFileName}_results/Called/*tab""")
    path("""${genomeFileName}_results/Position_Tables/*tab""")
```
Would you like change one of them to make it the same on both modules?

Reply:
Absolutely, the `mtb_mono` one is the correct one 👍

Reply:
But while making these changes please also make sure that the rest of the directives have the same content.

Reply:
yes, sure, I'll update it accordingly on this PR!